types/src/error.rs

```diff
@@ -48,7 +48,7 @@ impl<T: fmt::Display> fmt::Display for Mismatch<T> {
 pub enum CallError {
 	/// Invalid params in the call.
 	#[error("Invalid params in the call")]
```

Comment:
you want to change this to `#[error(transparent)]` or `#[error("Invalid params in the call: {0}")]`

Reply:
Good catch, ty!